lib/dash/dash_parser.js

```diff
@@ -94,6 +94,13 @@ shaka.dash.DashParser = class {
      */
     this.contextCache_ = new Map();
 
+    /**
+     * @private {
+     *   !Map<string, {endTime: number, timeline: number, reps: Array<string>}>
+     * }
+     */
+    this.continuityCache_ = new Map();
```

Comment:
We never cleared this cache, so it's a possible memory leak, we need to fix this.

Reply:
Also we need clear it on stop()

Reply:
Completely forgot to clear it in stop. I'll take a look about removing old data in it for live content, but it doesn't fill up too quickly.

Reply:
@gkatsev Do you have any update?

Reply:
sorry, got busy with some other things at work, but it's on my list to get back to!

Reply:
I pushed a change for this. Verified with https://livesim2.dashif.org/livesim2/segtimelinenr_1/periods_60/timesubswvtt_en/testpic_6s/Manifest.mpd that it keeps the cache with only the currently active periods after it finishes processing.
I'll take a look at the conflict next, hopefully it is minor.